english/udacity/udacity.md

```diff
@@ -1,3 +1,5 @@
 Alas, there is not, you're stuck. Get Caroline on it!
 
-Click [here](https://github.com/cbuckey-uda) to visit Caroline's github page!
\ No newline at end of file
+Click [here](https://www.github.com/cbuckey-uda) to visit Caroline's github page!
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.